test/integration/testMaxLobSize.js

```diff
@@ -0,0 +1,265 @@
+const assert = require('assert');
+const async = require('async');
+const testUtil = require('./testUtil');
+const crypto = require('crypto');
+const snowflake = require('./../../lib/snowflake');
+const connOption = require('./connectionOptions');
+
+function generateRandomString(sizeInBytes) {
+  const bufferSize = Math.ceil(sizeInBytes / 2);
+  const buffer = crypto.randomBytes(bufferSize);
+  return buffer.toString('hex').slice(0, sizeInBytes);
+}
+
+describe.skip('Max LOB test', function () {
+  let connection;
+  const SMALL_SIZE = 16;
+  const ORIGIN_SIZE = 16 * 1024 * 1024; 
+  const MEDIEUM_SIZE = ORIGIN_SIZE * 2;
+  const LARGE_SIZE = MEDIEUM_SIZE * 2;
+  const MAX_LOB_SIZE = LARGE_SIZE * 2;
+
+  const singleTest = 'select randstr(50000000, 124);';
+  const tableName = 'my_lob_test';
+  const createTable = `create or replace table ${tableName} (c1 varchar, c2 varchar, c3 int)`;
+  const normalInsert = `insert into ${tableName}(c1, c2, c3) values `;
+  const bindInsert = `insert into ${tableName}(c1, c2, c3) values (?, ?, ?)`;
+  const positionalInsert = `insert into ${tableName}(c1, c2, c3) values (:2, :1, :3)`;
```

Comment:
This is named binding and the previous one with ? is positional binding

Reply:
Edited them.